### ISSUE TITLE
Monospace font in code blocks

### ISSUE THREAD

**Issue by Signum** [SeptemberHX/joplin-plugin-enhancement#30]
Is there a way to use a monospace (aka fixed-width) font in code blocks?

Example:

    ```sh
    john@overlord /usr/local/bin » LANG=C ls -al
    total 32
    drwxr-xr-x  2 root root 4096 Dez  21 23:12 .
    drwxr-xr-x 10 root root 4096 Jul  26  2022 ..
    -rwxr-xr-x  1 root root 7692 Nov  22  2021 apt
    -rwxr-xr-x  1 root root  536 Apr  26  2022 gnome-help
    -rwxr-xr-x  1 root root  196 Feb   7  2019 highlight-mint
    -rwxr-xr-x  1 root root  964 Mai  15  2019 search
    -rwxr-xr-x  1 root root  553 Apr  26  2022 yelp
    ```
Renders as:

![image](https://user-images.githubusercontent.com/36562/219204713-64b70458-afbd-4604-a013-eb3679373b09.png)

This messes up the rendered part.

(Otherwise this plugin is brilliant. Thanks so much.)

**Comment by SeptemberHX** [comment]
You can add the following css style in the `userchrome.css` file:

```css
.CodeMirror-linewidget.enhancement-code-block-marker-line-widget span,
.CodeMirror-linewidget.enhancement-code-block-marker-line-widget div {
    font-family: Cascadia Code !important;
}
```

Replace `Cascadia Code` with any fixed width font installed on your system.

**Comment by Signum** [comment]
Thanks for the hint @SeptemberHX. Turns out I had to rather use this CSS selector:

```
.enhancement-code-block-marker-line-widget div
{
    font-family: monospace !important;
}
```

This works as expected.

However the root cause seems to be this CSS definition:

![image](https://user-images.githubusercontent.com/36562/219978408-2baf26bd-63d5-4de0-a82a-a904f504632a.png)

Not sure where this comes from. But it overwrites whatever the user configured.
